modules/support-submitting-a-case-enable-approved-access.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-submitting-a-case-enable-approved-access_{context}"]
+= Submitting a support case to enable approved access for ROSA clusters
+
+.Procedure
+
+. Log in to the link:https://access.redhat.com/support/cases/#/case/list[*Customer Support*] page of the Red{nbsp}Hat Customer Portal.
+
+. Click *Get support*.
+
+. On the *Cases* tab of the *Customer Support* page:
+
+.. Optional: Change the pre-filled account and owner details if needed.
+
+.. Select the appropriate category for your issue: *Configuration*, and click *Continue*.
+
+. Enter the following information:
+
+.. In the *Product* field, select *{product-title}* or *{hcp-title-first}*.
+.. In the *Problem statement* field, enter *ROSA Approved Access*.
+.. Click *See more options*.
+
+. Select *OpenShift Cluster ID* from the drop-down.
+
+. Fill the remaining mandatory fields in the form:
+
+.. What are you experiencing? What are you expecting to happen?
+... Fill with *Approved Access*.
+
+.. Define the value or impact to you or the business.
+... Fill with *Approved Access*.
+.. Click *Continue*.
+
+. Select *severity* as *4(Low)* and click *Continue*.
```

Comment:
Just checking that "severity" really is lowercase in the UI, or if it should be capitalized?

Reply:
it was thanks